pom.xml

```diff
@@ -0,0 +1,82 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>se.sundsvall.dept44</groupId>
+        <artifactId>dept44-service-parent</artifactId>
+        <version>1.35</version>
+    </parent>
+    <groupId>se.sundsvall.workflow</groupId>
+    <artifactId>template-camunda</artifactId>
+    <version>1.0</version>
```

Comment:
Maybe use 0.1 as version number in our templates? Then we can lift to 1.0 when everything looks fine?